src/corelibs/U2Gui/src/options_panel/OPWidgetFactory.cpp

```diff
@@ -32,6 +33,16 @@ OPGroupParameters::OPGroupParameters(QString groupId, QPixmap headerImage, QStri
       groupDocumentationPage(documentationPage) {
 }
 
+/////////////////////////////////////////////////OPFactoryFilterVisitor/////////////////////////////////////////////////
+OPFactoryFilterVisitor::OPFactoryFilterVisitor(ObjectViewType _objectViewType,
```

Comment:
Anything of non-standard C++ type should be written as "const Type& _objectViewType"

Reply:
This isn't need in this PR, removed